tests/test_main_startup_rollback.py

```
@@ -0,0 +1,412 @@
+    make_patched_get_running_loop,
+)
+
+__all__ = [
+    "test_startup_rollback_cancels_check_connection_task",
+    "test_startup_rollback_removes_ready_file",
+    "test_startup_rollback_shutdowns_plugins_when_loaded",
+    "test_startup_rollback_stops_message_queue_when_started",
+    "test_startup_rollback_cleans_reconnect_state_and_callbacks",
+    "test_startup_rollback_closes_matrix_client",
+    "test_startup_rollback_closes_meshtastic_client",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)